Makefile

```
@@ -24,9 +25,16 @@ etcd-release: ## Interactively starts the release workflow.
 ##@ Helm dev targets - The etcd needs a copy of the targets from k8s.mk without image-import because we use a external image here.
 
 .PHONY: k8s-helm-etcd-apply
-k8s-helm-etcd-apply: ${BINARY_HELM} k8s-helm-generate $(K8S_POST_GENERATE_TARGETS) ## Generates and installs the helm chart.
+k8s-helm-etcd-apply: ${BINARY_HELM} helm-generate $(K8S_POST_GENERATE_TARGETS) ## Generates and installs the helm chart.
 	@echo "Apply generated helm chart"
 	@${BINARY_HELM} upgrade -i ${ARTIFACT_ID} ${K8S_HELM_TARGET}
 
 .PHONY: k8s-helm-etcd-reinstall
-k8s-helm-etcd-reinstall: k8s-helm-delete k8s-helm-etcd-apply ## Uninstalls the current helm chart and reinstalls it.
+k8s-helm-etcd-reinstall: helm-delete k8s-helm-etcd-apply ## Uninstalls the current helm chart and reinstalls it.
+
+.PHONY: k8s-helm-chart-import
+k8s-helm-chart-import: ${BINARY_HELM} k8s-generate helm-generate-chart helm-package-release ## Pushes the helm chart to the k3ces registry.
+	@echo "Import ${K8S_HELM_RELEASE_TGZ} into K8s cluster ${K3CES_REGISTRY_URL_PREFIX}..."
+	@${BINARY_HELM} push ${K8S_HELM_RELEASE_TGZ} oci://${K3CES_REGISTRY_URL_PREFIX}/k8s ${BINARY_HELM_ADDITIONAL_PUSH_ARGS}
+	@echo "Done."
+
```

```suggestion
helm-etcd-reinstall: helm-delete helm-etcd-apply ## Uninstalls the current helm chart and reinstalls it.

.PHONY: helm-etcd-chart-import
helm-etcd-chart-import: ${BINARY_HELM} k8s-generate helm-generate-chart helm-package-release ## Pushes the helm chart to the k3ces registry.
	@echo "Import ${K8S_HELM_RELEASE_TGZ} into K8s cluster ${K3CES_REGISTRY_URL_PREFIX}..."
	@${BINARY_HELM} push ${K8S_HELM_RELEASE_TGZ} oci://${K3CES_REGISTRY_URL_PREFIX}/k8s ${BINARY_HELM_ADDITIONAL_PUSH_ARGS}
	@echo "Done."

```
We could rename the targets to match the new naming-pattern from k8s-component.mk